### DISCUSSION TITLE
[@types/rdf-ext] Missing the reduce function

### DISCUSSION THREAD

**Discussion by lecoqlibre** [DefinitelyTyped/DefinitelyTyped#65000] [Issues with a @types package]
The reduce function is missing in the [Dataset.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/rdf-ext/lib/Dataset.d.ts).

The reduce function was added here: https://github.com/rdf-ext/rdf-ext/pull/99.

Also, it would be nice that the DatasetExt is exported from the index so we can import it with `import { DatasetExt } from 'rdf-ext` instead of `import { DatasetExt } from 'rdf-ext/lib/Dataset'`.

Thanks!

@tpluscode @jeswr

[a comment by an automated account is not shown]

**Comment by tpluscode** [comment]
Good catch @lecoqlibre. Please see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65001 and maybe check if it works in your code

**Comment by tpluscode** [comment]
I'm not sure we should export `DatasetExt` from the main module if it's not exported in `rdf-ext` itself. Maybe a type-only export...
